device/dell/x86_64-dell_z9100_c2538-r0/plugins/sfputil.py

```diff
@@ -1,60 +1,331 @@
-#!/usr/bin/env python
+# sfputil.py
+#
+# Platform-specific SFP transceiver interface for SONiC
+#
 
 try:
-    from sonic_sfp.sfputilbase import sfputilbase
-except ImportError, e:
-    raise ImportError (str(e) + "- required module not found")
-
-
-class sfputil(sfputilbase):
-    """Platform specific sfputil class"""
-
-    port_start = 0
-    port_end = 31
-    ports_in_block = 32
-
-    port_to_eeprom_mapping = {}
-    port_to_i2c_mapping = {
-        0: 18,
-        1: 19,
-        2: 20,
-        3: 21,
-        4: 22,
-        5: 23,
-        6: 24,
-        7: 25,
-        8: 26,
-        9: 27,
-        10: 28,
-        11: 29,
-        12: 31,
-        13: 30,
-        14: 33,
-        15: 32,
-        16: 34,
-        17: 35,
-        18: 36,
-        19: 37,
-        20: 38,
-        21: 39,
-        22: 40,
-        23: 41,
-        24: 42,
-        25: 43,
-        26: 44,
-        27: 45,
-        28: 46,
-        29: 47,
-        30: 48,
-        31: 49
-    }
-
-    _qsfp_ports = range(0, ports_in_block + 1)
-
-    def __init__(self, port_num):
-        # Override port_to_eeprom_mapping for class initialization
-        eeprom_path = '/sys/class/i2c-adapter/i2c-{0}/{0}-0050/eeprom'
-        for x in range(0, self.port_end + 1):
-            port_eeprom_path = eeprom_path.format(self.port_to_i2c_mapping[x])
-            self.port_to_eeprom_mapping[x] = port_eeprom_path
-        sfputilbase.__init__(self, port_num)
+    import time
+    from sonic_sfp.sfputilbase import SfpUtilBase
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+
+class SfpUtil(SfpUtilBase):
+    """Platform-specific SfpUtil class"""
+
+    PORT_START = 0
+    PORT_END = 31
+    PORTS_IN_BLOCK = 32
+    IOM_1_PORT_START = 0
+    IOM_1_PORT_END = 11
+    IOM_2_PORT_START = 12
+    IOM_2_PORT_END = 21
+    IOM_3_PORT_START = 22
+    IOM_3_PORT_END = 31
+
+    BASE_VAL_PATH = "/sys/class/i2c-adapter/i2c-{0}/{0}-003e/"
+
+    _port_to_eeprom_mapping = {}
+    _port_to_i2c_mapping = {
+           0: [9, 18],
+           1: [9, 19],
+           2: [9, 20],
+           3: [9, 21],
+           4: [9, 22],
+           5: [9, 23],
+           6: [9, 24],
+           7: [9, 25],
+           8: [8, 26],
+           9: [8, 27],
+           10: [8, 28],
+           11: [8, 29],
+           12: [8, 31], #reordered 
```

Comment:
Add extra space before inline comment (pep8 style asks for minimum two spaces before the inline comment begins), and add space between `#` and `reordered`.

Also remove trailing whitespace.